src/style.css

```diff
@@ -0,0 +1,23 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+}
+
+h1 {
+  font-size: 17px;
+  margin-top: 0;
+}
+
+h2 {
+  font-size: 15px;
+  font-weight: normal;
+  margin: 1em 0;
+}
+
+p {
+  margin: 0;
+}
+
+.linkGreen {
```

Comment:
`linkGreen` is a bad name for the class. It is not semantical and means nothing. You should call it either just 'link' or don't use classes at all, and just use 'a' selector for this task